js/ui/scheduler/ui.scheduler.work_space.js

```diff
@@ -445,6 +448,14 @@ var SchedulerWorkSpace = Widget.inherit({
         return this.option("intervalCount") > 1;
     },
 
+    _toggleWorkSpaceOverlappingClass: function() {
+        this.element().toggleClass(WORKSPACE_WITH_OVERLAPPING_CLASS, this._isWorkSpaceWithOverlapping());
+    },
+
+    _isWorkSpaceWithOverlapping: function() {
+        return !!this.invoke("getMaxAppointmentsPerCell");
```

Comment:
In case if `maxAppointmentsPerCell: "none"` result will be `true`. Is it expected?